src/main/java/com/minecolonies/coremod/util/RecipeHandler.java

```diff
@@ -83,12 +94,26 @@ private static void enableInDevelopmentFeatures(final boolean enable)
     {
         if (enable)
         {
-            GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockHutBlacksmith, 1), "XXX", "X#X", "XXX", 'X', PLANK_WOOD, '#', "ingotIron"));
-            GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockHutStonemason, 1), "XXX", "X#X", "XXX", 'X', PLANK_WOOD, '#',
-                    Blocks.STONEBRICK));
+            addHutRecipe(new ItemStack(ModBlocks.blockHutBlacksmith, 1), "ingotIron");
+            addHutRecipe(new ItemStack(ModBlocks.blockHutStonemason, 1), Blocks.STONEBRICK);
         }
     }
 
+    private static void addHutRecipe(@NotNull final ItemStack hutItemStack, @NotNull final Item item)
+    {
+        GameRegistry.addRecipe(new ShapedOreRecipe(hutItemStack, "XBX", "X#X", "XXX", 'B', ModItems.buildTool, 'X', PLANK_WOOD, '#', item));
+    }
+
+    private static void addHutRecipe(@NotNull final ItemStack hutItemStack, @NotNull final String item)
+    {
+        GameRegistry.addRecipe(new ShapedOreRecipe(hutItemStack, "XBX", "X#X", "XXX", 'B', ModItems.buildTool, 'X', PLANK_WOOD, '#', item));
+    }
+
+    private static void addHutRecipe(@NotNull final ItemStack hutItemStack, @NotNull final Block block)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "addHutRecipe" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)